_data-prepper/common-use-cases/sampling.md

```diff
@@ -0,0 +1,78 @@
+
+In the following example, only 20 percent of events with a status code of `200` are sent to the sink from a given IP address:
+
+```json
+...
+  processor:
+  - aggregate:                                                                                                                                          
+        identification_keys: ["clientip"]  
+        duration :                                                                                                    
+        action:                                                                                                                                           
+          percent_sampler:                                                                                                                                   
+            percent: 20                                                                                                                        
+        when: "/status == 200" 
+...
+```
+
+## Tail sampling
+
+Use the `tail_sampler` action within the `aggregate` processor to sample events based on a set of defined policies. This action waits for an aggregation to complete across different aggregation periods based on the configured wait period. When an aggregation is complete, and if it matches the specific error condition, it's sent to the sink. Otherwise, only a configured percentage of events is sent to the sink.
+
+The following pipeline sends all OpenTelemetry traces with an error condition status of `2` to the sink. It only sends 20 percent of the traces that don't match this error condition to the sink.
+
+```json
+...
+  processor:
+   - aggregate:                                                                                                                                          
+        identification_keys: ["traceId"]                                                                                                                   
+        action:                                                                                                                                           
+          tail_sampler:                                                                                                                                   
+            percent: 20                                                                                                                                   
+            wait_period: "10s"                                                                                                                            
+            condition: "/status == 2"                                                                                                              
+          
+...
+```
+
+If you set the error condition to `false` or don't include it, only the configured percentage of events is allowed to pass through, determined by a probabilistic outcome.
```

Comment:
The meaning of the end of this sentence is a bit unclear. Can we clarify what we mean re: "as determined by a probabilistic outcome"?

Reply:
@dlvenable Please see Nate's question above. Thank you.